lib/commands/docker.ts

```diff
@@ -3,7 +3,10 @@
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
 
-import { basename } from 'path'
+function basename(path: string): string {
+	return path.split(/[\\/]/).pop() ?? ''
```

Comment:
I expect we are only on POSIX so backslash is a valid part of a filename

```suggestion
	return path.split('/').pop()
```

(also the result of split will always be an array with at least one string element and the result of pop always that element, so the nullish operator is useless).

Reply:
> I expect we are only on POSIX so backslash is a valid part of a filename

Don't we have colleagues running Windows? But indeed, this could break if the filename contains `\`. Not sure how to best handle that.

> (also the result of split will always be an array with at least one string element and the result of pop always that element, so the nullish operator is useless).

Typing was not happy. I added `as string` instead.